lib/routes/scut/gzic/notice.ts

```diff
@@ -0,0 +1,87 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+
+const categoryMap = {
+    xsyg: { title: '学术预告', tag: '30284' },
+    jytz: { title: '教研通知', tag: '30307' },
+    hwxx: { title: '海外学习', tag: 'hwxx' },
+    swtz: { title: '事务通知', tag: '30283' },
+};
+
+export const route: Route = {
+    path: '/gzic/notice/:category?',
+    categories: ['university'],
+    example: '/scut/gzic/notice/swtz',
+    parameters: { category: '通知分类，默认为 `swtz`' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '广州国际校区 - 通知公告',
+    maintainers: ['gdzhht'],
+    handler,
+    description: `| 学术预告 | 教研通知 | 海外学习 | 事务通知 |
+  | -------- | -------- | -------- | -------- |
+  | xsyg     | jytz     | hwxx     | swtz     |
+    :::warning
+    由于学校网站对非大陆 IP 的访问存在限制，可能需自行部署。
+    部分通知详情页可能会被删除（返回 404），或在校园网外无法访问。
+    :::`,
```

Comment:
```suggestion

:::warning
由于学校网站对非大陆 IP 的访问存在限制，可能需自行部署。
部分通知详情页可能会被删除（返回 404），或在校园网外无法访问。
:::`,
```
Add a line break between each markdown block

Indent description with 0 or 2 spaces.